tests/nightly/test_large_array.py

```diff
@@ -352,6 +352,93 @@ def test_topk():
     assert l.sum() == np.sum(np.arange(0, SMALL_Y))
 
 
+def test_add():
+    a = create_2d_tensor(rows=LARGE_X, columns=SMALL_Y, dtype=np.float64)
+    b = create_2d_tensor(rows=LARGE_X, columns=SMALL_Y, dtype=np.float64)
+    mx_res = a.__add__(b)
+    np_res = b.asnumpy()+a.asnumpy()
```

Comment:
nit: add space around operator +

Reply:
Shouldn't this be caught by the linter? I think we are not linting this folder. There are similar whitespace issues down.

Reply:
I ran `make lint` and `make pylint` before pushing this commit.